scripts/bootstrap.sh

```diff
@@ -169,8 +169,8 @@ cleanup() {
 }
 
 pycmd() {
-	[[ ${DEBUG} = "1" ]] && echo /usr/bin/python -c "$@" > /dev/stderr
-	/usr/bin/python -c "$@"
+	[[ ${DEBUG} = "1" ]] && echo /usr/bin/python3 -c "$@" > /dev/stderr
+	/usr/bin/python3 -c "$@"
```

Comment:
Do we still need this python change for Flatcar?
Also it is hard to know it is a Flatcar patch, as it does not have comments, only commit messages.

Reply:
I suppose we can try dropping this. This should work if we do the bootstrap from the newish version of Flatcar (so stable or newer, as LTS 2022 still has python2 as the default).

Reply:
Dropped python3 modification. I'll rerun the CI to see if it works.